usr/src/cmd/fs.d/nfs/svc/nlockmgr

```diff
@@ -60,5 +60,16 @@ then
 		echo "$0: WARNING svcadm refresh failed" 1>&2
 	fi
 fi
-   
+
+#
+# We have to wait for statd to finish starting up before lockd can
+# start running. If statd hangs after service startup (so SMF thinks
+# it's done) but before it registers an rpc address, we can end up
+# failing in the kernel when we attempt to contact it.
+#
+until /usr/bin/rpcinfo -T tcp 127.0.0.1 status >/dev/null 2>&1
```

Comment:
what we will do in zone where we have this service, but have no network configured?
we will be in long term loop?